www_src/blocks/image.jsx

```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+var React = require("react");
+
+var Image = React.createClass({
+  statics: {
+    defaults: {
+      src: '',
+      alt: '',
+      opacity: 1,
+      borderWidth: 0,
+      borderColor: 'transparent'
+    }
+  },
+  getDefaultProps: function () {
+    return this.defaults;
```

Comment:
`Image` is undefined at this point, also it appears statics get attached to instances as well, so what's the downside?